adam/tests/integration_tests/backwards_propagation_test.py

```diff
@@ -0,0 +1,93 @@
+from adam import Service
+from adam import Batch
+from adam import PropagationParams
+from adam import OpmParams
+from adam import BatchRunManager
+from adam import ConfigManager
+
+import json
+import unittest
+import datetime
+import os
+
+import numpy as np
+import numpy.testing as npt
+
+class BatchRunnerTest(unittest.TestCase):
+    """Integration test of batch running.
+    
+    """
+    def setUp(self):
+        config = ConfigManager(os.getcwd() + '/test_config.json').get_config()
```

Comment:
May I have the `test_config.json` file to test out locally? Thanks!

Reply:
Sent!

As far as the failing test, I believe it's detecting a genuine problem - since the number of projects grew so much, app engine can't actually list them before timing out. I'm working on fixing it on the server side (unfortunately not an easy fix) and would be happy to either put this in as an accurate statement of the state of our system or wait till all's in a good state (I suppose the latter would be slightly preferable so that messed up server side doesn't obscure problems with the tests)